src/Microsoft.NET.Sdk.Functions.MSBuild/Tasks/DeployStatus.cs

```diff
@@ -0,0 +1,12 @@
+﻿namespace Microsoft.NET.Sdk.Functions.MSBuild.Tasks
+{
+    public enum DeployStatus
```

Comment:
Is there a public webpage that denotes these various deploy status values?

Reply:
I couldn't fine one. I had to check the source to track it down.